### PR TITLE
Add dockerfiles for three nebula services

### PR DESCRIPTION
**Pull request by yixinglu**
- [X] Add nebula `graphd/metad/storaged` dockerfile for production evironment
- [X] Decrease docker image size
- [X] Don't run service in daemon method
- [x] Start to build `nightly` image when PR merged
- [x] Add `.dockerignore`
- [x] Rename base rpm package #928 

Move graphd/metad/storaged Dockerfile to [this repo: `nebula-docker-compose`](https://github.com/vesoft-inc/nebula-docker-compose)

close #838

### CODE REVIEW

File: Dockerfile.graphd [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+COPY . /home/nebula/BUILD
+
+RUN cd /home/nebula/BUILD/package \
+  && . /etc/profile.d/devtoolset-8.sh \
+  && ./make_srpm.sh -v $(git rev-parse --short HEAD) -p /home/nebula graphd
+
+FROM centos:7
+
+COPY --from=builder /home/nebula/RPMS/x86_64/nebula-graphd-*.el7.x86_64.rpm /usr/local/nebula/nebula-graphd.rpm
```
Comment (laura-ding) [inline Dockerfile.graphd]: we need the common rpm, it is nebula-*.el7.centos.x86_64.rpm

File: Dockerfile.metad [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+COPY . /home/nebula/BUILD
+
+RUN cd /home/nebula/BUILD/package \
+  && . /etc/profile.d/devtoolset-8.sh \
+  && ./make_srpm.sh -v $(git rev-parse --short HEAD) -p /home/nebula metad
+
+FROM centos:7
+
+COPY --from=builder /home/nebula/RPMS/x86_64/nebula-metad-*.el7.x86_64.rpm /usr/local/nebula/nebula-metad.rpm
```
Comment (laura-ding) [inline Dockerfile.metad]: ditto

File: Dockerfile.storaged [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+COPY . /home/nebula/BUILD
+
+RUN cd /home/nebula/BUILD/package \
+  && . /etc/profile.d/devtoolset-8.sh \
+  && ./make_srpm.sh -v $(git rev-parse --short HEAD) -p /home/nebula storaged
+
+FROM centos:7
+
+COPY --from=builder /home/nebula/RPMS/x86_64/nebula-storaged-*.el7.x86_64.rpm /usr/local/nebula/nebula-storaged.rpm
```
Comment (laura-ding) [inline Dockerfile.storaged]: ditto

File: Dockerfile.graphd [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+WORKDIR /usr/local/nebula
+
+RUN rpm -ivh nebula-graphd.rpm \
+  && mkdir -p ./{logs,data,pids} \
+  && rm -rf nebula-graphd.rpm
+
+EXPOSE 3369
```
Comment (laura-ding) [inline Dockerfile.graphd]: add Http server port

File: Dockerfile.metad [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+WORKDIR /usr/local/nebula
+
+RUN rpm -ivh nebula-metad.rpm \
+  && mkdir -p ./{logs,data,pids} \
+  && rm -rf nebula-metad.rpm
+
+EXPOSE 45500 45501
```
Comment (laura-ding) [inline Dockerfile.metad]: ditto

File: Dockerfile.storaged [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+WORKDIR /usr/local/nebula
+
+RUN rpm -ivh nebula-storaged.rpm \
+  && mkdir -p ./{logs,data,pids} \
+  && rm -rf nebula-storaged.rpm
+
+EXPOSE 44500 44501
```
Comment (laura-ding) [inline Dockerfile.storaged]: ditto

File: Dockerfile.graphd (hunk 4 above)
Comment (yixinglu) [reply Dockerfile.graphd]: Good catch!

File: Dockerfile.graphd (hunk 1 above)
Comment (yixinglu) [reply Dockerfile.graphd]: We want to run single service in one container. So only need to copy the specified rpm file.

Comment (laura-ding) [reply Dockerfile.graphd]: The three services have something in common, such as a shared script, they are packed together in nebula-${version}.el7.centos.x86_64.rpm

Comment (yixinglu) [reply Dockerfile.graphd]: Got it！

**Pull request merged**